src/fasta-recs.py

```diff
@@ -11,7 +12,50 @@ def main():
     )
     args = argparser.parse_args()
 
-    print(f"Now I need to process the records in {args.fasta}")
+    # Read the content
+    content = args.fasta.read()
+    
+    index = 0
+
+    while(index<len(content)):
+        # Look for '> *'
+        match = re.search(">(\s)*", content[index:])
+        if match is None:
+            break
+        index += match.end()
+
+        beginName = index
+
+        # Look for the end of the name
+        match = re.search("\n", content[index:])
+        if match is None:
+            chain = content[beginName:len(content)]
+            chain = re.sub('(\s)+',' ',chain).strip()
+            print(chain, end="\t")
+            break
+        index += match.start()
+
+        chain = content[beginName:index]
+        chain = re.sub('(\s)+',' ',chain).strip()
+        print(chain, end="\t")
+
+        beginChain = index        
+        # Look for the genome chain
+        match = re.search(">", content[index:])
+        #print(beginChain)
+        if match is None:
+            chain = content[beginChain:len(content)]
+            chain = re.sub('\n','', chain) # Delete intros
+            chain = re.sub('(\s)+',' ',chain).strip() # Replace spaces
+            print(chain)
+            break
+        index += match.start()
+        
+        # Delete spaces and intros
+        chain = content[beginChain:index]
+        chain = re.sub('\n','', chain) # Delete intros
```

Comment:
Newlines are not introns. Please don't confuse the poor reader (which right now is me) with misleading comments ;)

Also, isn't a newline just a space for regular expressions? Did you need to first remove newlines and then all other spaces? (might be an option, it depends from regular expression engine to engine, but I think mostly '\n' is a space just as much as ' ' or '\t').